models/resources.ts

```diff
@@ -53,3 +55,7 @@ export type ServicesList = CreateServicesListRequest & {
   taxonomies?: string[]
   createdAt: string
 }
+
+export const RESOURCE_SORT_METHODS = ['Last Modified', 'Created Time'] as const
```

Comment:
I think we discussed this a little during our last meeting but I feel like users might also want to sort by some fields alphabetically? Also ascending and descending for each sort field?

Reply:
added "title" as a ResourceSortMethod and a little button to switch between Ascending and Descending (still does 'Last Modified' descending by default). Not sure if that's the best way to do it though/whether my use of the "TriangleDownIcon" is appropriate.
![image](https://github.com/MutualAidNYC/services-lists/assets/4817361/a37c0f78-0af4-43e7-9b78-a52fd9abebd2)

Reply:
also I rebased it onto main and now I'm getting a typescript error from useHookFormProps (upgraded next and everything seems to be running fine locally though)

Reply:
Having an ascending / descending toggle button makes sense to me! Maybe give the button a `rotate` prop and / or a label to indicate whether the sort order is ascending or descending?

Reply:
added a title (assume that's what you meant by label) that changes based on sort order as well as a change from ChevronsUp to ChevronsDown depending on the state as well. I set the label to tell what the button does, not the current state, though not sure if that's intuitive.

Reply:
I also changed the sort function a bit so that it casts everything to lowercase and limits to alphanumeric characters. Given the form of the datetime strings the two time fields should have the same order, and I thought it made more sense for the titles since I was getting some weird sort orders based off what I think are things like spaces or quotation marks in the title.

Reply:
Also I might be getting a little bit tunnel visioned on the sorting stuff, but was curious if it made sense to add the "Created At" and "Last Modified" info to the resource displays given that we're letting users sort by them (I feel like a lot of the reason it's confusing is that for those fields you can't actually see the values you're sorting by).

Reply:
By label I just meant a some text to the button so if the actual component prop is `title` sure (but running locally myself this doesn't seem like the case). Imo it's more intuitive to have the label / title indicate what the state but looks like that's what you're doing already?

Tbh I'm not sure if / how non-alphanumeric characters affect JavaScript's sort function but if limiting gives you the behavior you're expecting I'm fine with it. If we wanted to be more thorough, we could write a generic sort function in another PR and write some tests for that (with / without non-alphanumeric characters).

Good callout! Imo I think it's helpful to add the `Created At` and `Last Modified` to the resource displays. Whether or not you wanna do that in this PR or another is up to you.

Reply:
oo by title I meant the little info blurb you get when you hover over the button. I think I was thinking a bit like editing in place where hovering over the state indicates you wanna know what the button does, but that might just not be noticeable at all.

Also added the Created Time and Last Modified to the resource card (re-requested review on that).
![image](https://github.com/MutualAidNYC/services-lists/assets/4817361/9aea287b-b3ed-46c7-bb34-5e4f8d7c0d2d)